pkgs/development/compilers/ghc/9.6.6-binary.nix

```diff
@@ -0,0 +1,386 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  perl,
+  gcc,
+  ncurses5,
+  ncurses6,
+  gmp,
+  libiconv,
+  numactl,
+  libffi,
+  llvmPackages,
+  coreutils,
+  targetPackages,
+
+  # minimal = true; will remove files that aren't strictly necessary for
+  # regular builds and GHC bootstrapping.
+  # This is "useful" for staying within hydra's output limits for at least the
+  # aarch64-linux architecture.
+  minimal ? false,
```

Comment:
Please remove all code relating to `minimal`, it doesn't really make sense to carry this around as this wouldn't even be built by hydra.nixos.org, so this is just unnecessary maintenance burden.